### PR TITLE
DOC-6607-Couchbase-Lite-2-7-1-Release-Notes

### PR DESCRIPTION
**Pull request by ibsoln**
https://issues.couchbase.com/browse/DOC-6607

This epic comprises the following tickets: 

- DOC-6608-Java 2.7.1 Rel Notes - https://issues.couchbase.com/browse/DOC-6608
- DOC-6609-Android 2.7.1 Rel Notes - https://issues.couchbase.com/browse/DOC-6609
- DOC-6610-Net 2.7.1 Rel Notes - https://issues.couchbase.com/browse/DOC-6610
- DOC-6611-ObjC 2.7.1 Rel Notes - https://issues.couchbase.com/browse/DOC-6611
- DOC-6612-Swift 2.7.1 Rel Notes - https://issues.couchbase.com/browse/DOC-6612

### CODE REVIEW

**Comment by ibsoln** [comment]
@rajagp @djpongh @amarantha-k  Couchbase Lite 2.7.1 Release Notes for review etc.

Staged here: https://ibsoln.github.io/stage/DOC-6607/couchbase-lite/2.7/index.html

**Review by amarantha-k — APPROVED** [review]
lgtm

**Review by djpongh — COMMENTED** [review]
Need to add:
+ * https://issues.couchbase.com/browse/CBL-849[*CBL-849*] Returning WebProxy null when WINHTTP_PROXY_INFO out as invalid value if that ever happens (for .NET). I added comments per file

File: modules/ROOT/pages/csharp.adoc [hunk 1]
```diff
@@ -1091,6 +1091,29 @@ The Couchbase Lite API is thread safe except for calls to mutable objects: `Muta
...
 == Release Notes
 
+=== 2.7.1
+
+Note: Support notices remain in force for this maintenance release -- see <<support-notices, Support Notices>> below.
+
+==== {fixed}
+
+This maintenance release fixes the following issues:
+
```
Comment (djpongh) [inline modules/ROOT/pages/csharp.adoc:1101]: Add https://issues.couchbase.com/browse/CBL-849 to .NET fixed section

**Comment by ibsoln** [comment]
> Need to add:
> 
> * * [https://issues.couchbase.com/browse/CBL-849[*CBL-849*]](https://issues.couchbase.com/browse/CBL-849%5B*CBL-849*%5D) Returning WebProxy null when WINHTTP_PROXY_INFO out as invalid value if that ever happens

@djpongh Is that just for C#/.Net or across the board?

File: modules/ROOT/pages/java-android.adoc [hunk 2]
```diff
@@ -1180,6 +1180,31 @@ The Couchbase Lite API is thread safe except for calls to mutable objects: `Muta
...
 
+=== 2.7.1
+
+Note: Support notices remain in force for this maintenance release -- see <<support-notices, Support Notices>> below.
+
+==== {fixed}
+
+This maintenance release fixes the following issues:
+
+* https://issues.couchbase.com/browse/CBL-789[*CBL-789*] Crash when accessing `connection->name()`
```
Comment (djpongh) [inline modules/ROOT/pages/java-android.adoc:1192]: Needs: https://issues.couchbase.com/browse/CBL-799

File: modules/ROOT/pages/java-platform.adoc [hunk 3]
```diff
@@ -1679,9 +1679,30 @@ Back to <<Preparing Your Build Environment>>
...
 == RELEASE NOTES
 
-=== .
+=== 2.7.1
 
-== 2.7.0
+==== {fixed}
+
+This maintenance release fixes the following issues:
+
```
Comment (djpongh) [inline modules/ROOT/pages/java-platform.adoc:1687]: Needs: https://issues.couchbase.com/browse/CBL-799

**Review by djpongh — COMMENTED** [review]
Missed two issues, one was accidentally marked private (made it public) and the other has two milestones on it.

**Comment by ibsoln** [comment]
> Missed two issues, one was accidentally marked private (made it public) and the other has two milestones on it.

@djpongh @amarantha-k  Post-review fixes applied - ready to re-review.

Commit: https://github.com/couchbase/docs-couchbase-lite/pull/206/commits/14c813f5b0377d024d578c9111f0c28bfcd2816f
Staged: https://ibsoln.github.io/stage/DOC-6607-post-review/couchbase-lite/2.7/index.html

**Review by djpongh — APPROVED** [review]

**Pull request merged**